src/styles/block/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  justify-content: center;
+  align-items: center;
+
+  &__image {
+    display: flex;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__image-content {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+    margin: 40px 16px 0;
+    padding: 0;
+
+    &-text {
+      color: $primary-color;
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 18px;
+      text-align: left;
+      margin-block: 0;
+    }
+  }
+
+  &__code {
+    display: flex;
+    margin-left: 16px;
+    margin-top: 4px;
+    height: 14px;
+    align-items: center;
+
+    &-text {
+      color: $secondary-color;
+      font-size: 10px;
+      line-height: 14px;
+      text-align: left;
+      font-weight: inherit;
+    }
+  }
+
+  &__price {
+    display: flex;
+    margin-top: 24px;
+    margin-inline: 16px;
+    justify-content: space-between;
+
+    &-text {
+      color: $secondary-color;
+      font-size: 12px;
+      font-weight: inherit;
+      line-height: 18px;
+      text-align: left;
+    }
+
+    &-value {
+      color: $primary-color;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+    }
+  }
+
+  &__button {
+    display: flex;
+    box-sizing: border-box;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
+    height: 40px;
+    margin: 16px;
+    border-radius: 5px;
+    background-color: $blue-color;
+
+    &-buy {
+      background-color: inherit;
+      border: none;
+      text-transform: uppercase;
+      color: $white-color;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      border-radius: 5px;
+      width: 100%;
+      height: 100%;
+      text-align: center;
+
+      &:hover {
+        color: $blue-color;
+      }
+    }
+
+    &--hover:hover {
+      background-color: $white-color;
+      border: 1px solid $blue-color;
+    }
```

Comment:
The '.card__button--hover:hover' selector modifies the background and border color on hover, but it's not clear if this is intended for the entire button or just a modifier. If it's for the entire button, the naming could be misleading, and it should be '.card__button:hover'. If it's a modifier, it's not being applied to any HTML element in the provided code.